test/libsolidity/semanticTests/viaYul/conditional/conditional_tuple.sol

```diff
@@ -0,0 +1,11 @@
+contract A {
+	function f(bool cond) public pure returns (uint, uint) {
+		(uint a, uint b) = cond ? (1, 2) : (3, 4);
```

Comment:
Nice - supporting this would have been a major hassle before the IRVariable PR :-).

Reply:
why?

Reply:
Well, there's no tuple assignments in Yul, so instead of just writing ``assign(conditional, leftExpression)`` (which was introduced with the IRVariables) you would have had to open up a case distinction for tuples and assign all components individually...